File: GFG/Arrays/006. Cyclically rotate an array by one/solution.py
def rotate(arr, n):
    last = arr[n - 1]
    for i in range(len(arr) - 1, 0, -1):
        arr[i] = arr[i - 1]

    arr[0] = last
    return arr


print(rotate([21, 22, 23, 24, 25], 5))
# print(rotate([1, 2]))

File: GFG/Arrays/006. Cyclically rotate an array by one/solution.java
import java.util.Arrays;

class Compute {

  public void rotate(long arr[], long n) {
    long x = arr[arr.length - 1];
    for (int i = arr.length - 1; i > 0; i--)
      arr[i] = arr[i - 1];
    arr[0] = x;
  }
}
